### PR TITLE
feat: (REPLAT-7091) redesign dailyregister slice for medium breakpoint

### PR DESCRIPTION
**Pull request by pdimova**
https://nidigitalsolutions.jira.com/browse/REPLAT-7091

Screenshots:
Android
Screenshot_1565691971
Screenshot_1565696256

iOS
Simulator Screen Shot - iPad Air 2 - 2019-08-13 at 14 37 39

### CODE REVIEW

File: packages/edition-slices/src/slices/dailyregisterleadfour/index.js [hunk 1]
```diff
@@ -52,6 +56,56 @@ class DailyRegisterLeadFour extends Component {
...
+    const natureLogo = (
+      <Logo
+        imageUri="https://www.thetimes.co.uk/d/img/DUR-nature-80d36dd1cd.png"
+        ratio={1 / 1}
+        style={styles.imageWrapper}
+        type="nature notes"
+      />
+    );
+
+    const birthdayLogo = (
```
Comment (ValGeorgiev) [inline packages/edition-slices/src/slices/dailyregisterleadfour/index.js]: Please move logos outside of render functions, they shouldn't be rebuild every time we render.

Comment (ValGeorgiev) [reply packages/edition-slices/src/slices/dailyregisterleadfour/index.js]: Ratio could be 1

File: packages/edition-slices/src/tiles/tile-at/markup.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
```
Comment (ValGeorgiev) [inline packages/edition-slices/src/tiles/tile-at/markup.js]: Could we reuse this file from tile-s? Probably to move it in the shared folder

File: packages/edition-slices/src/tiles/tile-at/markup.web.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
```
Comment (ValGeorgiev) [inline packages/edition-slices/src/tiles/tile-at/markup.web.js]: Same here

File: packages/edition-slices/src/slices/dailyregisterleadfour/index.js (hunk 1 above)
Comment (pdimova) [reply packages/edition-slices/src/slices/dailyregisterleadfour/index.js]: The way the logo looks like depends on the breakpoint that is passed within the render method.

File: packages/edition-slices/src/tiles/tile-at/markup.js (hunk 2 above)
Comment (pdimova) [reply packages/edition-slices/src/tiles/tile-at/markup.js]: dropping tileAT

**Pull request merged**